### ISSUE TITLE
Always show voting in log activity

### ISSUE THREAD

**Issue by Bananeweizen** [cgeo/cgeo#6034]
I'm wondering if we should always show the voting stars in the logging activity, if a GC cache is logged. For People without enabled voting it would basically be the trigger for enabling it.

**Comment by Lineflyer** [comment]
Good idea, but clicking should bring them to a good explanation of what they need to do.

**Comment by okainov** [comment]
Mention #7182 

With GCVote approach, voting bar should NOT be visible all the time, because user needs to log in in gcvote in addition to gc.com
